RevenueCatUI/Views/LinTemplateViews/TierFeatureAndPackageView.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  FeatureAndPackageViewWrapper.swift
+//  RevenueCat
+//
+//  Created by Guillaume LAURES on 05/09/2024.
+//
+
+import RevenueCat
+import SwiftUI
+
+@available(iOS 15.0, macOS 12.0, tvOS 15.0, watchOS 8.0, *)
+struct TierFeatureAndPackageView<FeaturesView: View, PackagesView: View>: View {
+    
+    @Environment(\.horizontalSizeClass)
+    var horizontalSizeClass
+    
+    let tiers: [PaywallData.Tier: TemplateViewConfiguration.PackageConfiguration.MultiPackage]
+    let allPackages: [TemplateViewConfiguration.Package]
+    
+    @Binding
+    var selectedPackage: TemplateViewConfiguration.Package
+    @Binding
+    var selectedTier: PaywallData.Tier
```

Comment:
are those `@Binding`'s used? could it be a simple prop?

Reply:
You're right, I can remove the binding.